### PR TITLE
CI Fixes

### PR DESCRIPTION
**Pull request by lsh-0**

### CODE REVIEW

**Comment by giorgiosironi** [comment]
https://alfred.elifesciences.org/job/pull-requests-projects/job/loris-docker/job/PR-2/ was now triggered.

**Comment by giorgiosironi** [comment]
Added `Butlers` in https://github.com/elifesciences/loris-docker/settings/access to receive commit statuses.

File: build.sh [hunk 1]
```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+docker build --tag elifesciences/loris .
```
Comment (giorgiosironi) [inline build.sh]: usually we build `elifesciences/loris:${IMAGE_TAG}`

File: run.sh [hunk 2]
```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+set -e
+
+docker run \
+    --rm \
+    --name loris \
+    --publish 5004:5004 \
+    loris
```
Comment (giorgiosironi) [inline run.sh]: `elifesciences/loris:${IMAGE_TAG}` too

File: build.sh (hunk 1 above)
Comment (lsh-0) [reply build.sh]: went with `${IMAGE_TAG:-latest}` - this makes it usable without environment variables

**Comment by lsh-0** [comment]
merging so I can update the iiif-formula and have it pull rather than build

**Pull request merged**